manifests/stub/validate_addr.pp

```diff
@@ -0,0 +1,25 @@
+# Class: unbound::stub::validate_addr
+#
+# Validate address arg of unbound::stub
+#
+define unbound::stub::validate_addr {
+  $addr_arr = split($name, '@')
+
+  # check if the address arg is a valid ip address
+  if is_ip_address($addr_arr[0]) {
+
+    # multiple '@' signs invalid
+    if size($addr_arr) > 2 {
+      fail('unbound::stub: invalid address. too many at signs in it.')
```

Comment:
Since we're testing for multiple '@', I like to see the use of '@' in the readme somewhere to give an indication how its used.